node_example.js

```diff
@@ -0,0 +1,33 @@
+var config = {
```

Comment:
are there ways to provide a `default` config for the bindings or is this needed? how is this solved for python bindings?

Reply:
a little hard to tell because I don't see tests and can't find the documentation for it, but from what I can tell looking at the code, it looks like they create a `configure` method that needs to be called in python before you can route. Does that sound right @kevinkreiser